### PR TITLE
[charts/csm-authorization]: CSM Authorization v2

### PR DESCRIPTION
**Pull request by atye**
#### Is this a new chart?

No

#### What this PR does / why we need it:

CSM Authorization v2

#### Which issue(s) is this PR associated with:

https://github.com/dell/csm/issues/1281

#### Checklist:

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [x] Chart Version bumped
- [ ] Variables are documented in the chart README.md
- [x] Title of the PR starts with the chart name (e.g. `[charts_dir/mychartname]`) if applicable

### CODE REVIEW

File: charts/csm-authorization/Chart.yaml [hunk 1]
```diff
@@ -8,11 +8,11 @@ description: |
...
 dependencies:
-  - name: cert-manager
-    version: 1.11.0
-    repository: https://charts.jetstack.io
-    condition: cert-manager.enabled
-  - name: ingress-nginx
-    version: 4.0.19
-    repository: https://kubernetes.github.io/ingress-nginx
-    condition: ingress-nginx.enabled
+- name: cert-manager
```
Comment (shaynafinocchiaro) [inline charts/csm-authorization/Chart.yaml:11]: Can you fix linting on this file?

File: charts/csm-authorization/values.yaml [hunk 2]
```diff
@@ -11,10 +17,11 @@ cert-manager:
 authorization:
   # images to use in installation
   images:
-    proxyService: dellemc/csm-authorization-proxy:v1.11.0
-    tenantService: dellemc/csm-authorization-tenant:v1.11.0
-    roleService: dellemc/csm-authorization-role:v1.11.0
-    storageService: dellemc/csm-authorization-storage:v1.11.0
+    proxyService: dellemc/csm-authorization-proxy:v1.10.0
```
Comment (shaynafinocchiaro) [inline charts/csm-authorization/values.yaml:20]: Can we move to the v2.0.0-alpha tag?

File: charts/csm-authorization/Chart.yaml (hunk 1 above)
Comment (atye) [reply charts/csm-authorization/Chart.yaml:11]: Reverted cert-manager to 1.10.0 like we have been using.

File: charts/csm-authorization/values.yaml (hunk 2 above)
Comment (atye) [reply charts/csm-authorization/values.yaml]: Fixed.

**Review by shaynafinocchiaro — APPROVED** [review]

**Review by KerryKovacevic — APPROVED** [review]

**Review by bharathsreekanth — APPROVED** [review]

**Review by alikdell — APPROVED** [review]

**Pull request merged**